modules/base-realms/realm-microsoft/outputs.tf

```diff
@@ -0,0 +1,27 @@
+output "standard_client_id" {
+  value = module.standard_client.client_id
+}
+
+output "standard_client_secret" {
+  value     = module.standard_client.client_secret
+  sensitive = true
+}
+
+output "realm_id" {
+  value = module.realm.id
+}
+
+output "realm_name" {
+  value = module.realm.name
+}
+
+
+# It is unclear to me if we need these sandbox credentials for the sandbox login
```

Comment:
This block is just from entra l think, which proxies through sandbox -> dev that this client links